src/Components/Dropdowns/Dropdown.scss

```diff
@@ -35,7 +35,7 @@ $dropdown-item--padding-v: $cf-marg-a + $cf-border;
   position: absolute;
   width: 100%;
   left: 0;
-  z-index: 500;
+  z-index: 5000;
```

Comment:
If we make variables of the z-indexes across the application it may be easier to avoid layering issues.